shared/elements/gaia_confirm/style.css

```diff
@@ -72,10 +72,13 @@ p {
   word-wrap: break-word;
   margin: 1rem 0 0;
   padding: 1rem 1.5rem;
-  border-top: 0.1rem solid #686868;
   line-height: 3rem;
 }
 
+p:first-of-type {
+  border-top: 0.1rem solid #686868;
```

Comment:
> I found only 1 gaia-confirm with several <p> (apps/keyboard/settings.html), so maybe I'm regressing it here. I suppose I fix it instead but maybe I should check with them ?

Not a big deal, but I'd rather make them aware of this change, just in case :)